src/hooks/useEditVideoSheet.tsx

```diff
@@ -1,19 +1,34 @@
 import React, { useCallback, useContext, useEffect, useState } from 'react'
-import { Draft } from '@/hooks'
 import { Location } from 'history'
 import { absoluteRoutes } from '@/config/routes'
 import { useNavigate } from 'react-router-dom'
 import { useLocation, useMatch } from 'react-router'
 import { RoutingState } from '@/types/routing'
+import { useOverlayManager } from '@/hooks/useOverlayManager'
+import { createId } from '@/utils/createId'
+import { useDrafts } from '@/hooks/useDrafts'
+import { useActiveUser } from '@/hooks/useActiveUser'
+import { useVideo } from '@/api/hooks'
+import { InputFilesState } from '@/shared/components/MultiFileSelect/MultiFileSelect'
+import { createUrlFromAsset } from '@/utils/asset'
+
+export type EditVideoSheetTab = {
+  id: string
+  isDraft?: boolean
+  isFresh?: boolean
```

Comment:
🐟  
`isFresh` means `new` here, right? Maybe `isNew` will be a more appropriate name here :P?

Reply:
Sure, that works as well